### PR TITLE
chore(release): v10.27.0

### PR DESCRIPTION
**Pull request by andreancardona**
Release PR for v10.27.0

### CODE REVIEW

File: config/jest-config-carbon/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Change Log
```
Comment (joshblack) [inline config/jest-config-carbon/CHANGELOG.md]: Make sure to remove any CHANGELOGs that are generated

**Review by joshblack — CHANGES_REQUESTED** [review]

File: config/jest-config-carbon/CHANGELOG.md (hunk 1 above)
Comment (andreancardona) [reply config/jest-config-carbon/CHANGELOG.md]: yep I saw that 🤕

File: packages/components/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "10.27.0-rc.4",
+  "version": "11.0.0",
```
Comment (joshblack) [inline packages/components/package.json]:
```suggestion
  "version": "10.27.0",
```

File: packages/pictograms/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@carbon/pictograms",
   "description": "Pictograms for digital and software products using the Carbon Design System",
-  "version": "11.2.0-rc.1",
+  "version": "12.0.0",
```
Comment (joshblack) [inline packages/pictograms/package.json]:
```suggestion
  "version": "11.2.0",
```

File: packages/react/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components-react",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "7.27.0-rc.4",
+  "version": "8.0.0",
```
Comment (joshblack) [inline packages/react/package.json]:
```suggestion
  "version": "7.27.0",
```

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by joshblack — APPROVED** [review]

**Review by aledavila — APPROVED** [review]
Yaaay 🥳

**Pull request merged**